cm_api.py

```diff
@@ -381,6 +381,21 @@ def deploy_cluster_kerberos_client_config(self, cluster_name, host_ref_list):
                                                                         cluster_name),
                           data=host_ref_list).json()
 
+    def first_run_cluster_service(self, cluster_name, service_name):
+        """Run firstRun on a service from the cluster.This command prepares and starts the service.
```

Comment:
Nit: Add space after period between `cluster` and `This`

Reply:
Done.